app/pages/lab/workflow.cjsx

```diff
@@ -286,6 +286,24 @@ EditWorkflowPage = React.createClass
               <hr />
             </div>}
 
+          {if 'Gravity Spy Gold Standard' in @props.project.experimental_tools
+            <div>
+              <div>
+                <AutoSave resource={@props.workflow}>
+                  <span className="form-label">Gravity Spy Gold Standard</span><br />
+                  <small className="form-help">Notify a user how they've classified a Gold Standard subject.</small>
+                  <br />
+                  <label htmlFor="gravity_spy_gold_standard">
```

Comment:
`htmlFor` is not necessary here since the input is the child of the `label` element. If you do use `htmlFor`, it needs to match the id of the form input it is for. In this case, the input is missing the id.